### DISCUSSION TITLE
Query help: .or with multiple not null checks

### DISCUSSION THREAD

**Discussion by kennethacohen** [supabase/supabase#1248] [Questions]
Just not sure how to formulate this in the supabase format: just trying to query a table and get results where at least one of two columns is not null. I'm assuming there is no way for me to just write vanilla sql in Supabase, right?

In vanilla sql:

```sql
SELECT *
FROM books
WHERE (category_id IS NOT NULL OR region_id IS NOT NULL)
```

My attempt:
```js
await supabase
.from("books")
.select("*")
.or("category_id.neq.null,region_id.neq.null,")
```

**Comment by steve-chavez** [comment]
Try it like:

```js
await supabase
.from("books")
.select("*")
.or("category_id.not.is.null,region_id.not.is.null")
```

Related: https://supabase.io/docs/reference/javascript/is
